src/components/Filters/DisabledOperator/DisabledOperator.tsx

```diff
@@ -4,10 +4,11 @@ import styled from 'styled-components';
 
 import { Text } from '../../typography';
 import { TextSizes } from '../../typography/Text/Text.enums';
-import { FlexContainer } from '../../FlexContainer';
+import { Padbox } from '../../layout';
 import { getColor, getRadii, pxToRem } from '../../../utils';
 
-const Container = styled(FlexContainer)`
+const Container = styled(Padbox)`
```

Comment:
Please remove padding rule on line 15 and put 
```
paddingSize={SpaceSizes.md} 
paddingType={PaddingTypes.squish}
```
instead of `allignItems="center"` on line 20